app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,34 @@
+class VotesController < ApplicationController
+  before_action :get_vote, only: [:show, :edit, :update, :destroy]
+
+  def new
+    @vote = Vote.new
+  end 
+
+  def create
+    @vote = Vote.create(vote_params)
+  end
+
+  def edit;end
+
+  def update
+    @vote = Vote.create(vote_params)
```

Comment:
@GerryLarios 
It looks like you're creating a Vote instance here, rather than updating it.